phoenicis-engines/src/main/java/org/phoenicis/engines/EngineSettingsManager.java

```diff
@@ -91,12 +91,12 @@ private String createFetchScript(RepositoryDTO repositoryDTO) {
         script.append("(function () {\n");
         script.append("var settings = {};\n");
         for (CategoryDTO engine : categoryDTOS) {
-            final String engineId = engine.getId();
+            final String engineId = engine.getId().replaceAll("^.*\\.", "");
```

Comment:
It may be worth a thought to extract this pattern to a static final variable.

Reply:
That's true but I wouldn't know where to put that. Also, this code here should be changed anyways long term. I'm thinking of something like `getScripts("engine.*.settings.*)`.